.circleci/config.yml

```diff
@@ -138,6 +138,15 @@ installtorchcpu: &installtorchcpu
       python -c 'import torch; print("Torch version:", torch.__version__)'
       python -m torch.utils.collect_env
 
+
+installdetectrondeps: &installdetectrondeps
+  run:
+    name: Install opencv, vqa-maskrcnn-benchmark
+    command: |
+      python -m pip install opencv-python==4.2.0.34
+      git clone https://gitlab.com/vedanuj/vqa-maskrcnn-benchmark.git maskbench
```

Comment:
I would suggest depending on a fixed commit. Just in case, we make some changes for our projects, we don't want this to break down.